src/travix/Travix.hx

```diff
@@ -619,6 +668,9 @@ class Travix {
       case 'php': t.doPhp();
       case 'python': t.doPython();
       case 'cs': t.doCs();
+      case 'lua': 
+        if(haxeVersion < "3.3.0") die('lua requires haxe >= 3.3.0');
```

Comment:
Wouldn't it be preferable if the test were skipped? Then `all` could always add `lua` and you'd get your results on haxe versions where it's supported. What's your rationale for raising an error?